### DISCUSSION TITLE
Change HOST from static to configurable setting

### DISCUSSION THREAD

**Discussion by hryvinskyi** [php-perfect/ddev-intellij-plugin#366] [Ideas]
Im using remote docker (https://ddev.readthedocs.io/en/stable/users/topics/remote-docker/)
but plugin always use 127.0.0.1 host for db configuration
can we change this constant to a configuration?

**Comment by rfay** [comment]
Wow, I didn't know that anybody *ever* used that. I'd be interested in talking with you outside this issue about what you do with it and why and how it works for you. https://discord.com/invite/5wjP76mBJD

I'm not sure it's fair to ask this plugin to handle this very obscure use-case though.
